Remotion/Web/Test/App.config

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <configSections>
+    <section name="remotion.web" type="Remotion.Web.Configuration.WebConfiguration, Remotion.Web" />
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
+  </configSections>
+
+  <remotion.web xmlns="http://www.re-motion.org/web/configuration">
+      <executionEngine urlMappingFile="~/bin/UrlMapping.xml" functionTimeout="16" refreshInterval="5" defaultWxeHandler="~/WxeHandler.ashx" />
+      <wcag conformanceLevel="Undefined" />
+  </remotion.web>
+
+  <log4net debug="true">
+    <appender name="TraceAppender" type="log4net.Appender.TraceAppender, log4net">
+      <layout type="log4net.Layout.PatternLayout,log4net">
+        <param name="ConversionPattern" value="%d %-5p %c [%x] - %m%n" />
+      </layout>
+    </appender>
+    <appender name="RollingLogFileAppender" type="log4net.Appender.RollingFileAppender">
+      <param name="File" value="log-data\\log.txt" />
+      <param name="AppendToFile" value="true" />
+      <param name="MaxSizeRollBackups" value="10" />
+      <param name="MaximumFileSize" value="5MB" />
+      <param name="RollingStyle" value="Size" />
+      <param name="StaticLogFileName" value="true" />
+      <layout type="log4net.Layout.PatternLayout">
+        <!--  value="date [thread] level logger [NDC] - message newline-->
+        <param name="ConversionPattern" value="%d [%t] %-5p %c [%x] - %m%n" />
+      </layout>
+    </appender>
+    <logger name="Remotion.Web.ExecutionEngine">
+      <level value="DEBUG" />
+      <appender-ref ref="TraceAppender" />
+    </logger>
+    <root>
+      <level value="ERROR" />
+      <appender-ref ref="RollingLogFileAppender" />
+    </root>
+  </log4net>
+  
+  <appSettings>
+    <add key="ValidationSettings:UnobtrusiveValidationMode" value="None" />
+  </appSettings>
+
+  <system.web>
+    <pages validateRequest="false" enableEventValidation="false">
+      <controls>
+        <add tagPrefix="remotion" namespace="Remotion.Web.UI.Controls" assembly="Remotion.Web" />
+        <add tagPrefix="remotion" namespace="Remotion.Development.Web.UnitTesting.AspNetFramework" assembly="Remotion.Development.Web" />
+        <add tagPrefix="remotion" namespace="Remotion.Web.Test.Shared.Rendering" assembly="Remotion.Web.Test.Shared" />
+      </controls>
+    </pages>
+    <httpHandlers>
+      <add verb="*" path="*.wxe" type="Remotion.Web.ExecutionEngine.WxeHandler, Remotion.Web" />
+      <add verb="*" path="WxeHandler.ashx" type="Remotion.Web.ExecutionEngine.WxeHandler, Remotion.Web" />
+    </httpHandlers>
+    <httpRuntime requestValidationMode="2.0"/>
+    <customErrors mode="Off" />
+    <compilation defaultLanguage="c#" debug="true" />
```

Comment:
Please check, the web.config had some updates in this regard and I don't know what the CorefForms default values are for the targetframework of compilation. httpRuntime is TF=net5.0/auto anyway, but compilation may be a different beast.

Reply:
The CoreForms default values are in the `config/machine.config`. As such the default `targetFramework="5.0"`